doc/api/fs.md

```diff
@@ -4377,10 +4377,6 @@ the returned {fs.FSWatcher}.
 The `fs.watch` API is not 100% consistent across platforms, and is
 unavailable in some situations.
 
-The recursive option is only supported on macOS and Windows.
-An `ERR_FEATURE_UNAVAILABLE_ON_PLATFORM` exception will be thrown
-when the option is used on a platform that does not support it.
-
```

Comment:
Should we add anything to the docs about limitations of the Linux/POSIX implementation and/or mention that it doesn't use native file watching? (I'm OK if the answer is "Let's try not doing that first and do it only if people end up needing that information.")

Reply:
We should add something to the YAML metadata (which, at this time, is at lines 4320-4335 in this file in the PR branch) about this change.

Reply:
I'm adding YAML metadata with this pull request. Thanks for the review @Trott